### DISCUSSION TITLE
Have to calibrate after waking up from sleep mode

### DISCUSSION THREAD

**Discussion by d3athwolf** [Rem0o/FanControl.Releases#2766] [General]
I'm Not sure if this is an issue or just not a thing?
Pretty much I have to recalibrate all the fans each time I wake my computer up from sleep mode, I'm not sure on startup but think I do as well.
But having to do that every time is getting annoying, I'm not sure on how that would be fixed only thing I could think of is having a check in a curve if it goes over 50'C / 122'F that it checks fan controls for that curve and check to see if calibrated, if not then re-calibrate, if it is then just do nothing

as iv been caught out plenty of times now where iv forgotten to calibrate my GPU fans and only to find out GPU siting around 80'c and throttling fps

**Comment by Rem0o** [comment]
Why do you think you need to recalibrate every time? What tells you this?  Screenshot?

**Comment by d3athwolf** [comment]
Here is a screenshot as you can see GPU already at 60c, but fan hasn't even kicked in
https://kappa.lol/zEp8S

and this is a screenshot of GPU after calibrated
https://kappa.lol/J9nWK
https://kappa.lol/lOM8W

as you can see I need to recalibrate it so that it... well works, and I have to do this every time I wake my computer up

**Comment by d3athwolf** [comment]
So can confirm still need to recalibrate controls after reboot as well
